src/chainparams.cpp

```diff
@@ -846,6 +853,17 @@ class CRegTestParams : public CChainParams {
             }
         };
 
+        m_assumeutxo_data = MapAssumeutxo{
+            {
+                110,
+                {uint256S("0x533d91c2aee01848b86693c226da68b1ba3f47bf266d9082a32bb2df4dafd7d8"), 110},
```

Comment:
how did you get these values?

Reply:
The expected value is generated through the following call chain that starts from [validation_chainstatemanager_tests.cpp](https://github.com/kittywhiskers/dash/blob/febbdfd1dc73c0c97e2dfe1d178d312e42c4fa33/src/test/validation_chainstatemanager_tests.cpp#L268) and ends with [a hash comparison check](https://github.com/kittywhiskers/dash/blob/febbdfd1dc73c0c97e2dfe1d178d312e42c4fa33/src/validation.cpp#L5940-L5944) against hardcoded assumeutxo hashes:

`CreateAndActivateUTXOSnapshot` > `ActivateSnapshot` > `PopulateAndValidateSnapshot`

It's just a matter of running `./src/test/test_dash -t validation_chainstatemanager_tests -- DEBUG_LOG_OUT | grep bad` and taking the expected hash. Be careful not the take the expected hash generated from one of the tests where the results are expected to be a validation failure.

The hash value for height 210 was calculated similarly bt by modifying the test to mine 100 _more_ blocks.